testdata/src/a/a.go

```diff
@@ -94,4 +94,27 @@ func ok() {
 			Str("bar", "baz").
 			Int("n", 1),
 		).Send()
+
+	// custom object marshaller
+	f := &Foo{Bar: &Bar{}}
+
+	log.Info().Object("foo", f).Msg("")
```

Comment:
Can you add the same test case but doesn't dispatch with `Msg("")` somewhere here?
https://github.com/ykadowak/zerologlint/blob/ff0e607007334f0e91940b4dc8236a379c6fb9cb/testdata/src/a/a.go#L43-L48